Source/WTF/wtf/cocoa/NSURLExtras.mm

```diff
@@ -45,7 +45,7 @@ static BOOL readIDNAllowedScriptListFile(NSString *filename)
     if (!filename)
         return NO;
 
-    FILE *file = fopen([filename fileSystemRepresentation], "r");
+    FILE *file = fopen(filename.fileSystemRepresentation, "r");
```

Comment:
Ditto.

Reply:
Ok I am confused about this one because the * symbol has not moved in this line.